CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 # Changelog
 
 All notable changes to this project will be documented here.
+
+### v2.0.1
```

Comment:
nitpick but I'd make it 2.1.0 since it is feature addition

Reply:
updated